SCMOD-12302: Add integration test

DavidGerald-Milligan opened the pull request:
https://portal.digitalsafe.net/browse/SCMOD-12302

DavidGerald-Milligan committed:
Removed copy of elasticsearch JDK

DavidGerald-Milligan committed:
Removing jdk later, reset JAVA_HOME

DavidGerald-Milligan committed:
Moved removal of jdk folder

DavidGerald-Milligan committed:
Add integration test adding an index

DavidGerald-Milligan committed:
Merge branch 'master' into SCMOD-12302

buildmachine-sou-jenkins2 commented:
A developer build has not yet been created for this branch. Click [here](http://sou-jenkins2.swinfra.net:8080/job/Create-New-Build/parambuild/?GITHUB_SERVER=github.com&GITHUB_ORGANISATION=CAFapi&REPO_NAME=opensuse-elasticsearch7-image&BRANCH_NAME=SCMOD-12302) to go ahead and create the build...

buildmachine-sou-jenkins2 commented:
CI Build Link:
http://sou-jenkins2.swinfra.net:8080/job/CAFapi/job/CAFapi~opensuse-elasticsearch7-image~SCMOD-12302~CI

dermot-hardy committed:
Code review: Removed unneeded version number

dermot-hardy committed:
Code review: Resolve minor warning in log

dermot-hardy committed:
Code review: Added dependency on Commons Logging Bridge

The elasticsearch-rest-high-level-client module uses commons-logging but caf-logging-log4j2 it already a dependency and it uses log4j2.

dermot-hardy committed:
Code review: Made test class final

dermot-hardy committed:
Code review: Made utility method private

dermot-hardy committed:
Whitespace changes: Source --> Format
